src/core/openthread.cpp

```diff
@@ -703,6 +703,16 @@ void otSetStateChangedCallback(otStateChangedCallback aCallback, void *aContext)
     sThreadNetif->RegisterCallback(sNetifCallback);
 }
 
+uint32_t otGetPollPeriod()
+{
+	return sThreadNetif->GetMeshForwarder().GetPollPeriod();
+}
+
+void otSetPollPeriod(uint32_t aPollPeriod)
+{
+	sThreadNetif->GetMeshForwarder().SetPollPeriod(aPollPeriod);
+}
```

Comment:
If SetPollPeriod() happens before 'thread start', device would start send DataRequest immediately which should not be expected.